config/initializers/remote_authenticatable.rb

```diff
@@ -0,0 +1,61 @@
+require 'devise/strategies/authenticatable'
```

Comment:
If we're just defining classes in this file then I think we should move it in `lib/`

Reply:
RemoteAuthenticatable overrides specific behaviour in Devise and its something that devise looks for.  We had originally called it cognito_authenticatable but devise didn't behave properly.